python/tvm/relax/pipeline.py

```diff
@@ -194,6 +194,7 @@ def _pipeline(mod: tvm.ir.IRModule, _ctx: tvm.transform.PassContext) -> tvm.ir.I
 # global map of pre-built pipelines
 PIPELINE_MAP = {
     "zero": zero_pipeline,
+    "default": default_build_pipeline,
```

Comment:
Just a note here. Likely we will deprecate the pipeline here in the short future.